spec/core_functions/color/blackness.hrx

```diff
@@ -25,61 +36,127 @@ a {
   b: 0%;
 }
 
+<===> min/warning
+DEPRECATION WARNING: Individual color-channel functions defined globally or in the color module are deprecated in favor of the new color.channel() function.
+
+Recommendation: color.channel(white, blackness)
+
+  ,
```

Comment:
Seems likely that these warnings will at some point have links where people can get more info about the changes. One thing we will need to find out is whether there is a preference for us to add placeholder lines like "More info: See https://sass-lang.com/d/some-function"